### ISSUE TITLE
Heater/Melter combination doesn't recognise fuel once ore is in place

### ISSUE THREAD

**Issue by MongoTheElder** [Darkosto/SevTech-Ages#191]
Low priority (has existed for many versions):

If you put ore in the melter and then add fuel to the melter, the fuel is not recognized. If you add fuel to the heater after ore is in the melter, the progress bar on the ore in the melter moves by one tick and stops. Replacing the ore once fuel is in either slot fixes the problem

![2017-11-19_13 28 10](https://user-images.githubusercontent.com/29290246/32993952-03ab4784-cd2e-11e7-82ea-c1f61fc9fae0.png)

After adding fuel to the melter:
![2017-11-19_13 28 19](https://user-images.githubusercontent.com/29290246/32993956-0adfaa5e-cd2e-11e7-9967-251ebd266896.png)

After adding fuel to the heater:
![2017-11-19_13 28 38](https://user-images.githubusercontent.com/29290246/32993960-230e57ec-cd2e-11e7-9d2a-36b24b108509.png)

**Comment by Darkosto** [comment]
This one is for @KnightMiner

I've known about the issue but didn't have a chance to talk to him about it, maybe he can take a look if he's not too busy :)
